### DISCUSSION TITLE
Wireshark & VM Analyst

### DISCUSSION THREAD

**Discussion by FritzLott** [Security-Onion-Solutions/securityonion#5827] [Unsupported Versions] [answered]
Using Wireshark within the VM Analyst GUI does NOT work in my installation.  It comes up without finding any network interfaces.  It works wonderfully from CLI (sudo wireshark).  I have tried changing the permissions on dumpcap without success.  I have tried to incorporate sudo in the Gnome menu command without success.  Starting a bash script from the Gnome menu with sudo in the script does NOT work (echo -e "password\n" | sudo -S wireshark).  Something is amiss and it's definitely beyond my pay grade.

Would someone with the VM Analyst installed please try Wireshark to test whether it's just my installation?  Thank you.

**Comment by FritzLott** [comment, marked as answer]
I found the answer at https://wiki.wireshark.org/CaptureSetup/CapturePrivileges

sudo setcap cap_net_raw,cap_net_admin+eip /usr/sbin/dumpcap

**Comment by dougburks** [comment]
We do not recommend sniffing live traffic with Wireshark as it is inherently dangerous. We recommend using tools that are specifically designed for sniffing live traffic like stenographer or tcpdump. Security Onion uses stenographer by default to do full packet capture:
https://docs.securityonion.net/en/2.3/stenographer.html
